docs/specs/2018-04-03.md

```diff
@@ -786,4 +786,14 @@ Example:
 Filter: 
   Domain: ['www.taobao.com','www.tmall.com']
   
-```
\ No newline at end of file
+```
+
+#### Default Role
+
+The default role will be generated when the policy is specified or when the service is configured with attributes such as vpc and nas that explicitly require specific permissions.
+
+The reason for this design is:
+
+1. Because the generation role user needs to assign a large privilege to the sub-user, in unnecessary scenarios, we try not to generate the default role as much as possible to avoid the user's sub-user privilege is insufficient.
```

Comment:
1. Because the generation of role needs a large permission. In unnecessary scenarios, we try not to generate the default role as much as possible to avoid the user's sub-user privilege is insufficient.